media-video/cheese/files/cheese-3.34.0-buildfix.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/meson.build b/meson.build
+index a4775d44..9d1e5587 100644
+--- a/meson.build
++++ b/meson.build
+@@ -149,11 +149,7 @@ gnome_video_effects_dep = dependency(
+ # Recommend some runtime GStreamer plugins.
+ gst_inspect = find_program('gst-inspect-1.0', required: false)
+ if gst_inspect.found()
+-  foreach plugin: ['camerabin', 'vp8enc', 'webmmux']
+-    if run_command(gst_inspect, plugin).returncode() != 0
+-      warning(plugin + ' was not found. It needs to be installed before Cheese is run')
+-    endif
+-  endforeach
++  warning('due Gentoo sandbox, unable to check for runtime GStreamer plugin dependencies: camerabin, vp8enc, webmmux')
```

Comment:
Let's make this line `unable to check for runtime GStreamer plugin dependencies due to Gentoo\'s sandbox: camerabin, vp8enc, webmmux`